style.css

```diff
@@ -0,0 +1,118 @@
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+}
+
+.top-header {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+}
+
+.icons-section {
+  padding: 15px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.time {
+  margin-left: 10px;
+}
+
+.menu-icon {
+  margin-right: 10px;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5px 20px;
+}
+
```

Comment:
- [ ] As said by the previous reviewer your design is still not similar to Figma's design so kindly make changes to it.
- [ ] Please update font weight, font-family, and manage the spacing in the project as Figma design.
### your design
![hisss](https://user-images.githubusercontent.com/81727988/170510718-54d7329f-0585-4389-a20a-3b4cfea53c71.png)
### Figma design
![fgigg](https://user-images.githubusercontent.com/81727988/170510760-d5a70210-8411-4137-9d1a-f6f914dd2fd9.png)